index.php

```diff
@@ -51,9 +51,13 @@
 require_once "include/dba.php";
 
 if (!$install) {
-	dba::connect($db_host, $db_user, $db_pass, $db_data, $install);
+	$result = dba::connect($db_host, $db_user, $db_pass, $db_data);
```

Comment:
This is just another merge conflict your are summoning? Or is there a reason to drop the `install` parameter?

Reply:
I stumbled upon it as well, but it is okay. The "connect" now always returns "false" on an error. The later code then have to handle it by throwing this "system unavailable" message.

I hadn't checked by now if the return value is checked now on every place where there is a "connect". This - of course - had to be done as well.

Reply:
In this PR?

Reply:
You removed this "unavailable"-message in this PR. So it should be added to all places where the "connect" is called (except the installer of course).

Reply:
Fair enough.